boards/arm/frdm_kw41z/Makefile.board

```diff
@@ -2,4 +2,6 @@ DEBUG_SCRIPT = jlink.sh
 
```

Comment:
I like the idea, maybe something like OPENSDA_FW_TYPE={JLINK, MBED, ...} and use that to set things